scripts/src/submission/submission_test.py

```diff
@@ -305,7 +305,7 @@ class CertificationScenario:
         expected_is_valid_certification=False,
         expected_reason="[ERROR] Send OWNERS file by itself in a separate PR.",
     ),
-    # Invalid certification Submission contains OWNERS and report file, but ignore_owners is set to True
+    # Valid certification Submission contains OWNERS and report file, but ignore_owners is set to True
```

Comment:
Any reason why the comment's description of the test is inverting, but no code is changing? Just an error in the comment, but the test actually tests a valid condition?